org.kde.peruse.json

```diff
@@ -0,0 +1,442 @@
+                            "url": "https://github.com/gnustep/libs-base/releases/download/base-1_29_0/gnustep-base-1.29.0.tar.gz",
+                            "sha256": "fa58eda665c3e0b9c420dc32bb3d51247a407c944d82e5eed1afe8a2b943ef37",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 9105,
+                                "stable-only": true,
+                                "url-template": "https://github.com/gnustep/libs-base/releases/download/base-$version/gnustep-base-1.$patch.0.tar.gz"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "name": "unarchiver",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "cd XADMaster && make -j $FLATPAK_BUILDER_N_JOBS -f Makefile.linux",
+                        "install -Dm755 XADMaster/unar XADMaster/lsar -t /app/bin/"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://cdn.theunarchiver.com/downloads/TheUnarchiverSource.zip",
+                            "sha256": "ac4cedf9b49375c0f805bda010fa03629d05ab283a0a8e35bf2dd01ae477e0a1",
+                            "strip-components": 0
+                        },
+                        {
+                            "type": "shell",
+                            "commands": [
+                                "sed -s -i 's#/usr/include/GNUstep#/app/include#g' */Makefile.linux"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name": "discount",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "./configure.sh --prefix=/app --enable-all-features --with-fenced-code --shared",
+                        "sed -e 's|/sbin/ldconfig|/sbin/ldconfig -n|' -i librarian.sh",
+                        "make install"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/Orc/discount/archive/v3.0.0a.tar.gz",
+                            "sha256": "9a20af1ee09d2369543a421ba65e6fc1fd482ad3a5b1828132441f2c8dd98afc",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 12139,
+                                "stable-only": true,
+                                "url-template": "https://github.com/Orc/discount/archive/v$version.tar.gz"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "name": "boost",
```

Comment:
Seems boost is only needed by poppler and it is a only a make depends of poppler. Can you try cleaning it up completely and check if everything works?